server/src/instant/util/java.clj

```diff
@@ -0,0 +1,9 @@
+(ns instant.util.java
+  (:import
+   (clojure.lang Var$Unbound)
+   (java.lang AutoCloseable)))
+
+(defn close [o]
+  (when (and o (not (instance? Var$Unbound o)))
```

Comment:
Seems a bit odd to me to check if you're getting an unbound var in `close`. I'd expect the caller to check for that.

Reply:
Yes, this is mostly to simplify our pattern of doing (def) conditionally inside init function